components/project-loader/project-loader.jsx

```diff
@@ -144,4 +156,10 @@ export default React.createClass({
       </div>
     );
   }
-});
+}
+
+ProjectLoader.propTypes = {
+  featured: PropTypes.string.isRequired
```

Comment:
`featured` isn't a required `prop` so let's remove `isRequired`.

We should also check types for other `props` as well.

```js
ProjectLoader.propTypes = {
  featured: PropTypes.string,
  showCounter: PropTypes.bool,
  moderationState: PropTypes.object,
  bookmarkedOnly: PropTypes.bool,
  ids: PropTypes.array
};
```